tests/improver-sleet-probability/00-basic.bats

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bats
+# -----------------------------------------------------------------------------
+# (C) British Crown Copyright 2017-2019 Met Office.
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# * Redistributions of source code must retain the above copyright notice, this
+#   list of conditions and the following disclaimer.
+#
+# * Redistributions in binary form must reproduce the above copyright notice,
+#   this list of conditions and the following disclaimer in the documentation
+#   and/or other materials provided with the distribution.
+#
+# * Neither the name of the copyright holder nor the names of its
+#   contributors may be used to endorse or promote products derived from
+#   this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+. $IMPROVER_DIR/tests/lib/utils
+
+@test "sleet-probability" {
+  improver_check_skip_acceptance
+  KGO="sleet/basic/kgo.nc"
+
+  # Run remainder processing and check it passes.
+  run improver sleet-probability \
+      "$IMPROVER_ACC_TEST_DIR/sleet/basic/half_prob_snow_falling_level.nc" \
+      "$IMPROVER_ACC_TEST_DIR/sleet/basic/tenth_prob_snow_falling_level.nc" \
```

Comment:
I don't think these inputs represent how the code will be used. They definitely show the CLI doing the correct maths, but one contains values of 0 or 0.5 and the second contains values of 0 or 0.1 (with the 0.1 being colocated with the 0.5s), therefore the output KGO has values of 1 or 0.4. I'll talk to Katie to see if we can create more realistic CLI test inputs.

Reply:
I'm going to pick this up in a follow-on ticket when we have realistic input data for this CLI test.